src/components/Dissolution/CustodianOfRecords.vue

```diff
@@ -1,216 +1,211 @@
-                          filled
-                          label="First Name"
-                          id="person__first-name"
-                          v-model.trim="custodian.officer.firstName"
-                          :rules="isPerson ? Rules.FirstNameRules : []"
-                          @input="syncCustodianPartyType(PartyTypes.PERSON)"
-                        />
-                      </v-col>
-                      <v-col>
-                        <v-text-field
-                          filled
-                          class="px-5"
-                          label="Middle Name (Optional)"
-                          id="person__middle-name"
-                          v-model.trim="custodian.officer.middleName"
-                          :rules="isPerson ? Rules.MiddleNameRules : []"
-                          @input="syncCustodianPartyType(PartyTypes.PERSON)"
-                        />
-                      </v-col>
-                      <v-col>
-                        <v-text-field
-                          filled
-                          label="Last Name"
-                          id="person__last-name"
-                          v-model.trim="custodian.officer.lastName"
-                          :rules="isPerson ? Rules.LastNameRules : []"
-                          @input="syncCustodianPartyType(PartyTypes.PERSON)"
-                        />
-                      </v-col>
-                    </v-row>
-
-                    <!-- Org input -->
-                    <v-radio :value=PartyTypes.ORGANIZATION class="pt-2">
-                      <template slot="label">
-                        <span
-                          class="person-or-org-option"
-                          :class="{ 'title-label': isInError }"
-                        >
-                          Corporation or Firm Name
-                        </span>
-                      </template>
-                    </v-radio>
-                    <v-row no-gutters class="pt-4">
-                      <v-col>
-                        <v-text-field
-                          filled
-                          label="Corporation or Firm Name"
-                          id="organization__name"
-                          v-model.trim="custodian.officer.organizationName"
-                          :rules="isOrg ? Rules.OrgNameRules : []"
-                          @input="syncCustodianPartyType(PartyTypes.ORGANIZATION)"
-                        />
-                      </v-col>
-                    </v-row>
-                  </v-radio-group>
-
-                  <v-divider class="mt-n3 mb-8" />
-                </template>
-
-                <label>Email Address</label>
+    <!-- EDIT SECTION -->
+    <v-card v-if="!isSummary" flat class="edit-section rounded-4">
+      <v-form ref="addCustodianForm" class="add-custodian-form" v-model="addCustodianValid">
+        <v-row no-gutters>
+          <v-col cols="12" sm="2" class="pr-4 pb-4">
```

Comment:
Edit columns are 2 and 10.

The right padding is to have a gutter to force the label to wrap before running into the second column.

The bottom padding is to have some vertical whitespace between the label and the data in very small viewports (where each column is full width, ie. stacked).

Reply:
Columns are now 3 and 9 everywhere!